sdk/authorization/Microsoft.Azure.Management.Authorization/src/Generated/Models/RoleAssignment.cs

```diff
@@ -136,10 +136,11 @@ public RoleAssignment(string roleDefinitionId, string principalId, string id = d
         public string Description { get; set; }
 
         /// <summary>
-        /// Gets or sets the conditions on the role assignment. This limits the
-        /// resources it can be assigned to. e.g.:
+        /// Gets or sets the conditions on the role assignment. An Azure role assignment condition is an additional check that you can optionally add to your role assignment to provide more fine-grained access control.
```

Comment:
Where does this change come from? I don't see any metadata change here. If you just manually modified the description here, this is not allowed. You should try to update your swagger spec first and then the SDK.